doc/connectivity/bluetooth/api/index.rst

```diff
@@ -40,3 +37,44 @@ Bluetooth APIs
    shell/mcp.rst
    shell/tmap.rst
    shell/pbp.rst
+
+Bluetooth LE Host
+=================
+
+.. toctree::
+   :maxdepth: 1
+
+   services.rst
+   gap.rst
+   gatt.rst
+   att.rst
+
+Bluetooth Mesh
+==============
+
+.. toctree::
+   :maxdepth: 1
+
+   mesh.rst
+
+Core Host and drivers
+=====================
+
+.. toctree::
+   :maxdepth: 1
+
+   l2cap.rst
+   connection_mgmt.rst
+   data_buffer.rst
+   hci_drivers.rst
+   hci_raw.rst
+   crypto.rst
+
+Other
+=====
+
+.. toctree::
+   :maxdepth: 1
+
+   controller.rst
+   uuid.rst
```

Comment:
Is there any specific order of these?

Previously it looks like they were sorted by file name, but now that seems random. 

Not sure if it matters at all

Reply:
I tried to have them in stack top-down order